docs/public/api/GettingStartedWithLogs.md

```diff
@@ -0,0 +1,63 @@
+# Getting Started with Logs
+
+## Setup
+
+1. Install the `Microsoft.Extensions.Logging` package
+
+    ```sh
+    dotnet add package Microsoft.Extensions.Logging
+    ```
+
+1. Update `Program.cs` with the following
+
+    ```c#
+    using Microsoft.Extensions.Logging;
+    using OpenTelemetry.Logs;
+
+    public class Program
+    {
+        public static void Main()
+        {
+            using var loggerFactory = LoggerFactory.Create(builder =>
+            {
+                builder.AddOpenTelemetry(options => options
+                    .AddConsoleExporter());
+            });
+
+            var logger = loggerFactory.CreateLogger<Program>();
+            logger.LogInformation("Hello from {name} {price}.", "tomato", 2.99);
+        }
+    }
+    ```
+
+1. Run the application
+
+    ```sh
+    dotnet run
+    ```
+
+1. You should see the following output
+
+    ```text
+    LogRecord.TraceId:            00000000000000000000000000000000
+    LogRecord.SpanId:             0000000000000000
+    LogRecord.Timestamp:          2020-11-13T23:50:33.5764463Z
+    LogRecord.EventId:            0
+    LogRecord.CategoryName:       Program
+    LogRecord.LogLevel:           Information
+    LogRecord.TraceFlags:         None
+    LogRecord.State:              Hello from tomato 2.99.
+    ```
+
+Congratulations! You are now collecting logs using OpenTelemetry.
+
+<!-- TODO Can we just import the Program.cs file (skip copyright would be nice too) -->
```

Comment:
Maybe a small script?

Reply:
Found out how to do this from Sphinx!
- https://www.jetbrains.com/pycharm/guide/tutorials/sphinx_sites/documentation/
- https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-literalinclude